src/app/wizard/step/provider-settings/provider/extended/component.ts

```diff
@@ -47,13 +50,21 @@ export class ProviderExtendedComponent extends BaseFormValidator implements OnIn
   readonly Providers = NodeProvider;
   readonly Controls = Controls;
 
+  // This is only supported on AWS, Azure, GCP, and Openstack
+  supportsAllowedIPRangeOverride =
+    this.Providers.AWS || this.Providers.AZURE || this.Providers.GCP || this.Providers.OPENSTACK;
+
   constructor(private readonly _builder: FormBuilder, private readonly _clusterSpecService: ClusterSpecService) {
     super('Provider Extended');
   }
 
   ngOnInit(): void {
     this.form = this._builder.group({
       [Controls.ProviderExtended]: this._builder.control(''),
+      [Controls.NodePortsAllowedIPRange]: this._builder.control(CIDR_ALLOW_ALL, [
```

Comment:
Is it really a required property?

Reply:
It shouldn't be empty if this is enabled

Reply:
I've asked if it is a required property meaning that a value has to be provided in order to proceed further.

Reply:
Logic has been modified for this, PTAL.